mlir/tools/mlir-miopen-driver/mlir-miopen-driver.cpp

```diff
@@ -37,6 +37,8 @@
 #include "llvm/Support/ToolOutputFile.h"
 #include "llvm/Support/raw_ostream.h"
 
+#include "hip/hip_runtime.h"
```

Comment:
`mlir-miopen-driver` shouldn't be dependent to HIP. Please put HIP runtime dependency codes to `mlir-rocm-runner` and upstream them.